### PR TITLE
Fix #4072: jupyterhub slack link

### LINKED ISSUE #4072: Add Slack link to JupyterHub
On JupyterHub pages, we want a prominent link to Slack.

### PR DESCRIPTION
**Pull request by gurhar1133**
* There is now a slack icon link in jupyterhub on development jupyterhub
* The next steps will involve making changes to rsconf/.../conf.py.jinja for production
Thoughts @robnagler @e-carlin

### CODE REVIEW

File: sirepo/package_data/jupyterhub_templates/page.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        </span>
+    </a>
+</li>
+<script>
+const SIREPO = {};
+</script>
+<script src="/auth-state">
+</script>
+<script>
+    const slackUri = SIREPO.authState.slackUri;
```
Comment (robnagler) [inline sirepo/package_data/jupyterhub_templates/page.html:16]: No need for temporary variable.

File: sirepo/package_data/jupyterhub_templates/page.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <a id="sirepo-slack-link" href="https://slack.sirepo.com">
+         <span>
+             <img style="filter: invert(0.5);" width="70" src="/static/svg/slack.svg" title="Join us on Slack"/>
+        </span>
+    </a>
+</li>
+<script>
+const SIREPO = {};
+</script>
+<script src="/auth-state">
```
Comment (robnagler) [inline sirepo/package_data/jupyterhub_templates/page.html:13]: Put a jinja comment `{# #}` with a POSIT for where this is defined. Unfortunately, you can't put comments in JSON so you can't POSIT in schema-common.json.

**Review by robnagler — CHANGES_REQUESTED** [review]

File: sirepo/jupyterhub.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 from __future__ import absolute_import, division, print_function
 from pykern import pkconfig
 from pykern.pkcollections import PKDict
+import pykern.pkresource
```
Comment (e-carlin) [inline sirepo/jupyterhub.py:10]: sort

File: sirepo/package_data/jupyterhub_templates/page.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{% extends "templates/page.html" %}
+{% block nav_bar_right_items %}
+<li>
+    <a id="sirepo-slack-link" href="https://slack.sirepo.com">
```
Comment (e-carlin) [inline sirepo/package_data/jupyterhub_templates/page.html:4]: We use 2 space indent in html

**Comment by gurhar1133** [comment]
Changes made

**Review by robnagler — APPROVED** [review]

**Pull request merged**